tests/test_project/templates/test_project/console_log_test.html

```
@@ -0,0 +1,22 @@
+    <title>Console log capture test</title>
+    {# Emitted while the page is still parsing, before readyState === "complete", #}
+    {# to verify console logs produced during page load are captured. #}
+    <script>
+      console.log("page-load-marker");
+    </script>
+  </head>
+  <body>
+    <div id="main-content">console log capture test</div>
+    {# A secondary (iframe) browsing context that navigates after the top-level #}
+    {# page has logged, to verify its navigation does not clear the buffer. #}
+    <iframe id="console-log-test-frame"></iframe>
```

HTMLHint (1.9.2)

[warning] A <iframe> element must have an accessible name.

(frame-title-require)